homeassistant/components/apple_tv.py

```diff
@@ -91,7 +91,7 @@ def configuration_callback(callback_data):
         hass.async_add_job(configurator.request_done, instance)
 
     instance = configurator.request_config(
-        hass, 'Apple TV Authentication', configuration_callback,
+        'Apple TV Authentication', configuration_callback,
```

Comment:
I'm actually super confused. If configurator was already pointing at `hass.components.configurator`, how come that it would even work with `hass` being passed in? 🤔  or has this been broken.